src/scrapers/base-scraper.ts

```diff
@@ -56,7 +56,9 @@ export class BaseScraper<TCredentials extends ScraperCredentials> implements Scr
       const success = scrapeResult && scrapeResult.success === true;
       await this.terminate(success);
     } catch (e) {
-      scrapeResult = createGenericError((e as Error).message);
+      if (!(e as Error).message.includes('Target.closeTarget')) {
```

Comment:
You're hiding the error here like you sure that error is not flagging we are doing something wrong.

Even if so, I want you to catch it and hide it inside the terminate function because it related to it.